lib/model/unittest/CEventRateModelTest.cc

```diff
@@ -41,7 +36,6 @@
 
 #include "CModelTestFixtureBase.h"
 
-#include <boost/foreach.hpp>
 #include <boost/range.hpp>
```

Comment:
You don't necessarily have to do it in this PR, but just an observation that this is here for `boost::size` applied to various types of container.  The loops that use it can probably be changed to range based for loops, and then this header can go too.

Also, while checking that I noticed there's a stray `boost::shared_ptr` in this file that wasn't changed to `std::shared_ptr` for some reason.